tests/algorithm_validation_tests/exaflow/conftest.py

```diff
@@ -0,0 +1,20 @@
+import pytest
```

Comment:
All of these should be moved to the `algorithms` folder.

The `algorithm_validation_tests` is only for prod algorithms and validating their precision.